test/xspec.bats

```diff
@@ -455,41 +461,40 @@ teardown() {
 }
 
 @test "invoking xspec.sh using -catalog with spaces in file path uses XML Catalog resolver" {
-    mkdir cat\ a\ log
-    mkdir cat\ a\ log/xspec
-    cp catalog/catalog-01* cat\ a\ log
+    space_dir="${work_dir}/cat a log"
+    mkdir -p "${space_dir}/xspec"
+    cp catalog/catalog-01* "${space_dir}"
+    
     export SAXON_CP="$SAXON_CP:$XML_RESOLVER_CP"
-	run ../bin/xspec.sh -catalog cat\ a\ log/catalog-01-catalog.xml cat\ a\ log/catalog-01-xslt.xspec
+	run ../bin/xspec.sh -catalog "${space_dir}/catalog-01-catalog.xml" "${space_dir}/catalog-01-xslt.xspec"
 	echo "$output"
 	[ "$status" -eq 0 ]
 	[ "${lines[7]}" = "passed: 1 / pending: 0 / failed: 0 / total: 1" ]
-	rm -rf cat\ a\ log
 }
 
 @test "invoking xspec.sh using XML_CATALOG with spaces in file path uses XML Catalog resolver" {
-    mkdir cat\ a\ log
-    mkdir cat\ a\ log/xspec
-    cp catalog/catalog-01* cat\ a\ log
+    space_dir="${work_dir}/cat a log"
+    mkdir -p "${space_dir}/xspec"
+    cp catalog/catalog-01* "${space_dir}"
+    
     export SAXON_CP="$SAXON_CP:$XML_RESOLVER_CP"
-    export XML_CATALOG=cat\ a\ log/catalog-01-catalog.xml
-	run ../bin/xspec.sh cat\ a\ log/catalog-01-xslt.xspec
+    export XML_CATALOG="${space_dir}/catalog-01-catalog.xml"
+	run ../bin/xspec.sh "${space_dir}/catalog-01-xslt.xspec"
 	echo "$output"
 	[ "$status" -eq 0 ]
 	[ "${lines[7]}" = "passed: 1 / pending: 0 / failed: 0 / total: 1" ]
-	rm -rf cat\ a\ log
 }
 
 @test "invoking xspec.sh using SAXON_HOME finds Saxon jar and XML Catalog Resolver jar" {
-    export SAXON_HOME="${PWD}/saxon"
-    mkdir $SAXON_HOME
-    cp $SAXON_CP $SAXON_HOME
-    cp $XML_RESOLVER_CP $SAXON_HOME/xml-resolver-1.2.jar
+    export SAXON_HOME="${work_dir}/saxon"
+    mkdir "${SAXON_HOME}"
+    cp "${SAXON_CP}"        "${SAXON_HOME}"
```

Comment:
Very minor issue: I think the whitespace between `SAXON_CP` and `SAXON_HOME` could be removed.

Reply:
Addressed in #302.